org/ystia/docker/containers/generic/playbooks/create.yaml

```
@@ -16,20 +19,48 @@
   - name: Get python major version
     set_fact:
       python_major_version: "{{pri.python_version | replace('\n', '') | regex_replace('^(\\d+).*', '\\1') }}"
-  - name: Install pip version compatible with python 2
-    easy_install:
-      name: pip<21.0
+  - name: Set python version for pip
+    set_fact:
+      python_pip_pkg: "python3-pip"
+      pip_cmd: "pip3"
+    when: python_major_version != "2"
+  - name: Set python 2 version for pip
+    set_fact:
+      python_pip_pkg: "python-pip"
+      pip_cmd: "pip"
     when: python_major_version == "2"
-  - name: Install pip
-    easy_install:
-      name: pip
-      state: latest  
+  - name: RedHat - install prerequisites
+    yum:
+      name:
+        - "{{ python_pip_pkg }}"
+      state: present
+      update_cache: yes
+    when: ansible_os_family == 'RedHat'
+  - name: Debian - install prerequisites
+    apt:
+      name:
+        - "{{ python_pip_pkg }}"
+      state: present
+      update_cache: yes
+    when: ansible_os_family == 'Debian'
```

If you have 2 containers that runs in // apt will likely fail on one instance as long as the first one haven't finish it's install.

```suggestion
    register: apt_res
    retries: 5
    delay: 15
    until: apt_res is success
    when: ansible_os_family == 'Debian'
```

---

ok!